common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -12,6 +12,7 @@
 from functools import reduce
 
 import six
+from django.contrib.auth.models import User
```

Comment:
pylint doesn't like this, should I be using from django.contrib.auth import get_user_model or something else?